src/conn/conn_dhandle.c

```diff
@@ -727,7 +727,7 @@ __wt_conn_dhandle_discard(WT_SESSION_IMPL *session)
 
 		WT_WITH_DHANDLE(session, dhandle,
 		    WT_TRET(__wt_conn_dhandle_discard_single(
-		    session, true, F_ISSET(conn, WT_CONN_IN_MEMORY))));
+		    session, true, false)));
```

Comment:
FTR, now that `__wt_conn_btree_sync_and_close` is checking `WT_CONN_IN_MEMORY` at https://github.com/wiredtiger/wiredtiger/pull/3512/files#diff-1724c58a241ea44b64a43d04fb861d5bR223, this change lgtm.